lib/publiccloud/provider.pm

```diff
@@ -466,6 +466,24 @@ sub terraform_destroy {
     record_info('ERROR', 'Terraform exited with ' . $ret, result => 'fail') if ($ret != 0);
 }
 
+sub terraform_param_tags
+{
+    my ($self) = @_;
+    my $tags = {
+        openqa_ttl        => get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300),
+        openqa_var_JOB_ID => get_current_job_id(),
+        map { 'openqa_var_' . $_ => get_var($_) } qw(NAME TEST VERSION FLAVOR DISTRI BUILD),
```

Comment:
can we at least drop NAME ? because it is basically combination of all other variables + jobid which we also have ?

Reply:
the side effect of this will be bigger records for each instance in PCW ...

Reply:
> can we at least drop NAME ? because it is basically combination of all other variables + jobid which we also have ?

we do not have the architecture :)

> the side effect of this will be bigger records for each instance in PCW ...

We live in times of big-data, lets play that game!

Ok, what about of dropping all and keep only `NAME`. My idea is, using it as the tooltip for the `openqa` link in pcw.
I also find it good, to double check if the jobid belongs to the expected openqa instance.

Reply:
> 
> We live in times of big-data, lets play that game!

sure no problem , but let's play it in sane order :) first proper handling of tags in PCW UI than throw tons of tags into VMs :) 

> 
> Ok, what about of dropping all and keep only `NAME`. My idea is, using it as the tooltip for the `openqa` link in pcw.
> I also find it good, to double check if the jobid belongs to the expected openqa instance.

works for me :+1: